MQTT_server.md

```diff
@@ -0,0 +1,33 @@
+# Introduction
+The structure of this information deisseminating network is based on MQTT protocol which consists of three components. `Connectivity Endpoint`, `tangle-accelerator` and `Parser`
+
+### 1. Connectivity Endpoint
+This is a client implemented with any possible MCU (or Raspberry Pi) and a communication module (i.e. NB-IoT module). There are some modems allowing users to send MQTT message with AT-command-like command, so for these kinds of modems, they don't need to run a `mosquitto`-dependent programs on the MCU.
```

Comment:
Change "MCU (or Raspberry Pi)" to "lightweight computing node (such as Raspberry Pi)".